src/components/common/Seo.tsx

```diff
@@ -17,6 +17,7 @@ export const Seo = ({
 
   return (
     <NextSeo
+      useAppDir={true}
```

Comment:
We will remove this component at all soon because we will switch to Next.js metadata ;)